src/miner.cpp

```diff
@@ -472,16 +473,19 @@ void BlockAssembler::addPackageTxs(int &nPackagesSelected, int &nDescendantsUpda
             }
         }
 
+        // Skip transactions that are under X seconds in mempool
+        if (iter->GetTime() > current_time - required_age_in_secs) {
```

Comment:
Shouldn't this be < ?

Reply:
no, I had it backwards first as well, which led to transactions only being accepted within first X seconds :)